CHANGELOG.md

```diff
@@ -11,7 +11,8 @@ Unreleased
 ----------
 
 ### Added
--   Adds French phoneme list and filtered tsv file. (\#213)
+-   Adds Serbo-Croatian phoneme list and filtered TSV files. (\#227)
```

Comment:
While 'we've not been super-consistent about this, I'd add to the end of the list, not the beginning.

Reply:
Okay, I think this should be fixed.